### DISCUSSION TITLE
Changing default "active tool size"

### DISCUSSION THREAD

**Discussion by sr258** [flameshot-org/flameshot#2861] [Q&A] [answered]
Hi,

Up until a few days ago, the "active tool size" in Flameshot was set to a reasonable default value. The arrows and lines were fairly slim and this is fine. Now the default value of "active tool size" is very large and arrows become really big:
![grafik](https://user-images.githubusercontent.com/15268740/183278440-36ee8ef8-1cb1-4b17-be4f-78ab177589e5.png)

I haven't found any way to change the "active tool size" default value. I'm not sure if I changed it accidentally or if the new setting is the new default. Anyway, I'd like to permanently change this value. Changing it in the menu only works for the current screenshot.

I use the Snap version of Flameshot on X11:
Flameshot v12.1.0 ()
Compiled with Qt 5.15.3
linux: 5.15.0-43-generic
ubuntu-core: 20

**Comment by mmahmoudian** [comment, marked as answer]
The answer have been right in front of your the whole time ;)

![image](https://user-images.githubusercontent.com/390889/183281143-1236fd05-c815-403e-831a-c932b4fe1ec4.png)

**Reply by ahmethtb** [reply]
Thanks! This was really helpful

**Reply by xcp-ustc** [reply]
You are such a genius！！！

**Comment by sr258** [comment]
Thanks a lot. Must have used the mouse wheel accidentally!

**Reply by mmahmoudian** [reply]
Just in case you haven't tested these already:

- you can change the tool size while you are drawing something. For example, while you are creating an arrow and still holding down the left mouse button, you can scroll to change the size
- you can change the size later for any object. Just deselect the active tool (if anything is active), then click on what you want to change. When selected, scroll
- you can change the size to an exact number just by typing it. For example try the following:
    1. select the arrow tool
    2. type `12` (note the size indicator at the top left of your monitor)
    3. draw an arrow

**Comment by hubyhuby** [comment]
My 2 cts, in a previous version it would remember the size of arrows I use. I think it was usefull, because most of the time I use the same arrows size... so now I have to do a little extra thig, use the mouse wheel... not a big deal. But would be nice to save between snapshot the previous size for example, or have a "save as default" in "Active tool size" left menu.

**Reply by mmahmoudian** [reply]
It should still remember the tool size. That part has not changed. Whoch version are you using, how did you install it, and what is your OS?

Also as a side-tip, you can select a tool, and type the value you want (between 1 to 99), and the size would change. I have explained multiple methods here:

https://github.com/flameshot-org/flameshot/discussions/2861#discussioncomment-3342434

**Reply by hubyhuby** [reply]
> It should still remember the tool size. That part has not changed. Whoch version are you using, how did you install it, and what is your OS?
> 
> Also as a side-tip, you can select a tool, and type the value you want (between 1 to 99), and the size would change. I have explained multiple methods here:
> 
> [#2861 (reply in thread)](https://github.com/flameshot-org/flameshot/discussions/2861#discussioncomment-3342434)

Thanks for getting back to me.
I was on linux mint latest, with default included Flamshot 12. Now I have just installe the flatpack 14.0.O  and I get the same behavior.
I have setup a workaround, I added the "+" button to increase each time my arrow. I am using so much this great tool any speed up is welcome in my workflow !
